content/community/gsoc/2021/ideas.html

```diff
@@ -753,33 +741,33 @@ <h4><img src="/images/App_Generic_32.png"/>
 
 <ul>
 <li>Skill set: general C/C++, userland development</li>
-<li>Possible mentors: </li>
+<li>Possible mentors: jua</li>
```

Comment:
you have reverted previous changes here. Jua is not available to mentor this year.